src/routes/auth.route.ts

```diff
@@ -2,6 +2,10 @@ import { Router } from 'express';
 import { forgotPassword, resetPassword, updatePassword } from '../controllers';
 import { AuthSchema, validateSchema } from '../validations/auth.validation';
 import { Login } from '../controllers/auth.controller';
+import passport from 'passport';
+import dotenv from 'dotenv';
+
+dotenv.config();
```

Comment:
We should only configure dotenv in the main entry app and other places we'll just tap into the process.env without configuration again. Please remove anywhere any of you might have repeated yourselves like this and make sure it's only called once and test that it works. This is the second time i've told you this
```suggestion
```